### PR TITLE
Fix code coverage

### PR DESCRIPTION
**Pull request by pjanotti**
Update tools and fix command, this gets code coverage working locally, not sure how it will interact with CI.

### Checklist
- [X] I ran Unit Tests locally.

### CODE REVIEW

**Comment by pjanotti** [comment]
Plan is to remove WIP status depending on how CI reacts to these changes.

File: .config/dotnet-tools.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (pjanotti) [inline .config/dotnet-tools.json:1]: Added this to pin a version of the tool.

File: build/Common.test.props [hunk 2]
```diff
@@ -23,16 +23,12 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
 
-  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
+  <PropertyGroup Condition="$(MSBuildProjectName.Contains('.Tests'))">
```
Comment (pjanotti) [inline build/Common.test.props:26]: There are test projects ending in `Win` perhaps I will change this to be any project under `test/**`.

**Comment by SergeyKanzhelev** [comment]
So this doc: https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/CONTRIBUTING.md#test is still up to date, correct?

**Comment by pjanotti** [comment]
> So this doc: https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/CONTRIBUTING.md#test is still up to date, correct?

Correct.

**Pull request merged**